cores/arduino/stm32/uart.c

```diff
@@ -126,6 +127,19 @@ void uart_init(serial_t *obj)
     return;
   }
 
+#ifdef STM32L4xx
+  // Select HSI as source clock 
```

Comment:
Please, Could you explain why only L4 and why only USART 1-3 ? Waht about UART4-5?